de-lib/src/main/java/org/iplantc/de/apps/integration/client/gin/AppEditorGinModule.java

```diff
@@ -8,10 +8,33 @@
 import org.iplantc.de.apps.integration.client.view.AppIntegrationPalette;
 import org.iplantc.de.apps.integration.client.view.AppsEditorView;
 import org.iplantc.de.apps.integration.client.view.AppsEditorViewImpl;
-import org.iplantc.de.apps.integration.client.view.propertyEditors.*;
+import org.iplantc.de.apps.integration.client.view.deployedComponents.cells.DCNameHyperlinkCell;
```

Comment:
I totally agree - I left it in case there was some preference for "Deployed Component" that I wasn't aware of.  I'll change in the future.